Initial HTMA (#184)
vatnz-dev/sops: docs/terminal/htma.md

<figure markdown>
  ![Hamilton TMA](./assets/hntma-airspace.png) 
  <figcaption>Hamilton TMA (CTA/D)</figcaption>
</figure>

### Transfer of Control Points

|Transfer Flow         | Requirements                                                  | Notes                                                                       |
| -------------------- | ------------------------------------------------------------- | --------------------------------------------------------------------------- |
| Tower -> Approach    | Once the aircraft becomes airborne off the runway             | Tower shall instruct all aircraft to contact HTMA airborne                  |
| Approach -> Control  | Once the aircraft are approaching the lateral bounds          | Aircraft may be handed off when nothing further is required of the aircraft |
| Control -> Approach  | Once the aircraft is approaching the cleared level by control | Aircraft may not always be cleared to A110 when handed to HTMA              |
| Approach -> Tower    | Over the IAF (RNP) or established inbound on the VOR/DME      |                                                                             |

## Departures

> HTMA may climb all domestic aircraft to thier RFL. International Flight may only be climbed to FL250, with further climb from ENR.

Reviewer: Misspelling of their on line 40.

HTMA may climb all domestic aircraft to **thier** RFL

---
  title: Hamilton Terminal Area
---

--8<-- "includes/abbreviations.md"

!!! note "Hamilton Procedures"
    The HTMA procedures build upon information already contained in the [NZHN Aerodrome Procedures](../aerodromes/Class-D/nzhn.md), and any Controller logged on to HTMA shall be familiar with those procedures.

## Positions

| Position Name     | Shortcode | Callsign              | Frequency | Login ID | Usage     |
| ----------------- | --------- | --------------------- | --------- | -------- | --------- |
| Hamilton TMA      | HTMA      | Bay Approach          | 125.300   | NZHN_APP | Secondary |

!!! note Hamilton TMA

>     Controllers shall note that HTMA is a **secondary** position therefore BAY must be staffed before a controller may log on HTMA. 

Reviewer 2: ```suggestion
    Controllers shall note that HTMA is a **secondary** position therefore BAY must be staffed (either directly or via inheritance/extended service) before a controller may log on HTMA. 
```
Not sure whether this is needed - does it make sense to expand do you think?

## Airspace

> Hamilton CTA/C follows the lateral and vertical boundaries as shown below. HTMA has a vertical limit of `A095`. Controllers shall be aware of which frequency to hand aircraft to, as shown on the image below.

Reviewer 2: Image doesn't really show any ownership (and hence frequency)?

Author: Have updated Airspace image to reflect the boundary

> Hamilton CTA/C follows the lateral and vertical boundaries as shown below. HTMA has a vertical limit of `A095`. Controllers shall be aware of which frequency to hand aircraft to, as shown on the image below.

Reviewer 2: ```suggestion
Hamilton CTA/D follows the lateral and vertical boundaries as shown below. HTMA has a vertical limit of `A095`. Controllers shall be aware of which frequency to hand aircraft to, as shown on the image below.
```

<figure markdown>
  ![Hamilton TMA](./assets/hntma-airspace.png) 
  <figcaption>Hamilton TMA (CTA/D)</figcaption>
</figure>

### Transfer of Control Points

|Transfer Flow         | Requirements                                                  | Notes                                                                       |
| -------------------- | ------------------------------------------------------------- | --------------------------------------------------------------------------- |
| Tower -> Approach    | Once the aircraft becomes airborne off the runway             | Tower shall instruct all aircraft to contact HTMA airborne                  |
| Approach -> Control  | Once the aircraft are approaching the lateral bounds          | Aircraft may be handed off when nothing further is required of the aircraft |

> | Control -> Approach  | Once the aircraft is approaching the cleared level by control | Aircraft may not always be cleared to A110 when handed to HTMA              |

Reviewer 2: ```suggestion
| Control -> Approach  | Once the aircraft is approaching the cleared level by control | Aircraft may not always be cleared to `A110` when handed to HTMA              |
```

| Approach -> Tower    | Over the IAF (RNP) or established inbound on the VOR/DME      |                                                                             |

## Departures

HTMA may climb all domestic aircraft to their RFL.

## International

> International Flights departing to the west via `UPLAR`, `MADEP` and `PEBLU` have a restriction of `A100` at `LEKUS`, this may be cancelled with coordination to RAN. All International departures may be climbed to FL250 by HTMA, with further climb from RAN.  

Reviewer 2: ```suggestion
International Flights departing to the west via `UPLAR`, `MADEP` and `PEBLU` have a restriction of `A100` at `LEKUS`, this may be cancelled with coordination to RAN. All International departures may be climbed to `FL250` by HTMA, with further climb from RAN.  
```

## Arrivals

>  All arrivals will be handed to HTMA prior to A110. HTMA may track shorten as required.

Reviewer 2: ```suggestion
 All arrivals will be handed to HTMA prior to `A110`. HTMA may track shorten as required.
```

>  HTMA shall note the `RNP Z` is CAT A/B/C/D only. While the `RNP Y` is CAT A/B only.

Reviewer 2: Maybe something more like this?
```suggestion
 HTMA shall note that whilst `RNP Z` is available for all RNP capable aircraft, `RNP Y` is CAT A/B only.
```

## Coordination

> HTMA shall coordinate any non-nominated approaches with HTWR. Any track shortening outside of HTMA's CTA shall be coordinated with the appropriate ENR sector.

Reviewer 2: ```suggestion
HTMA shall coordinate any non-nominated approaches with Hamilton Tower. Any track shortening outside of HTMA's CTA shall be coordinated with the appropriate enroute sector.
```
Just to better match with language used in other pages
